sources/packages/api/src/utilities/esdc-utils.ts

```diff
@@ -12,6 +12,18 @@ export function getMaritalStatusCode(
   return maritalStatus === RelationshipStatus.Single ? "S" : "O";
 }
 
+/**
+ * Gets the Part-Time marital status code conversion for the MSFAA request file.
+ */
+export function getPTMaritalStatusCode(
+  maritalStatus: RelationshipStatus,
+): string {
+  if (maritalStatus === RelationshipStatus.Married) {
+    return "MA";
+  }
+  return maritalStatus === RelationshipStatus.Single ? "SI" : "SP";
```

Comment:
Why we aren't using enum for marital status codes?

Reply:
I think, we already spoke about this I guess @dheepak-aot , these are just used for transformation.